ema_workbench/analysis/parcoords.py

```diff
@@ -143,6 +143,27 @@ class ParallelAxes:
     rot : float, optional
           rotation of axis labels
 
+
+    Attributes
+    ----------
+    limits : DataFrame
```

Comment:
Add a description of `limits`

Reply:
added the description from the parameters part